frontend/src/components/PhotoDetails.styles.js

```diff
@@ -0,0 +1,29 @@
+import styled from "styled-components";
+
+export const Wrapper = styled.div`
+display: flex;
+align-items: center;
+flex-direction: column;
+width: 1200px;
+max-width: 100%;
+margin: 0 auto;
+height: 100vh; //max-content
+`;
+
+export const Return = styled.button`
```

Comment:
A note for the future: creating global (but extendable) button styles might be useful.

Reply:
We have one button now.